### PR TITLE
Frontend login with JWT token support

### PR DESCRIPTION
**Pull request by AnsonCheng03**

### CODE REVIEW

**Comment by AnsonCheng03** [comment]
Note: you can right click inspect then delete the access token from local storage to remove the login state before the sign out page is done.

File: beaconnect-react-app/package.json [hunk 1]
```diff
@@ -33,16 +33,18 @@
...
     "@testing-library/user-event": "^13.2.1",
     "@types/jest": "^27.0.1",
     "@types/node": "^16.7.13",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
+    "graphql": "^16.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.10.0",
+    "react-router-dom": "6",
```
Comment (isaackhlam) [inline beaconnect-react-app/package.json:47]: What is the purpose of changing this version

File: beaconnect-react-app/src/index.tsx [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 import './index.css';
 import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
 root.render(
   <React.StrictMode>
-    <App />
+    <ApolloProvider client={client}>
+      <App />
+    </ApolloProvider>
```
Comment (isaackhlam) [inline beaconnect-react-app/src/index.tsx:13]: Kindly move this into App

File: beaconnect-react-app/src/pages/Login/components/login.query.ts
Comment (isaackhlam) [inline beaconnect-react-app/src/pages/Login/components/login.query.ts:1]: We have already define type for use, but change later, add to backlog.

**Review by isaackhlam — CHANGES_REQUESTED** [review]

File: beaconnect-react-app/package.json (hunk 1 above)
Comment (AnsonCheng03) [reply beaconnect-react-app/package.json:47]: It changes automatically when I install Apollo server, maybe due to compatibility issue

**Comment by isaackhlam** [comment]
Please don't resolve the conversation yourself before reviewed

**Review by isaackhlam — APPROVED** [review]
Add problem in backlog, go a head first

**Pull request merged**